### PR TITLE
[analyzer] Handle formatting path in drive different from cwd (#165).

### PR DESCRIPTION
**Pull request by ivanperez-keera**
The report-view command throws an exception when executed in Windows from a path in a drive different from that of the files being analyzed.

This is due to the use of `os.path.relpath` on two paths in potentially different drives as part of the definition of `format_path`. According to `relpath`'s definition, the function throws a `ValueError` on Windows if the two arguments provided are on different drives.

This PR modifies `format_path` so that it first checks if the paths are on different drives, returning the absolute path in that case, and executing the old behavior otherwise.

### CODE REVIEW

**Comment by ivanperez-keera** [comment]
@arthaud I'd be great if you can take a look, raise any comments regarding the patch or the commit message or approve it if all is ok.

After that, you can merge if you want, or I can.

Thanks!

**Comment by ivanperez-keera** [comment]
Minor change: I just modified the commit message. Unit is not a valid name for a drive in English :) All good, ready to review.

**Review by arthaud — APPROVED** [review]

**Pull request merged**
